leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/LeshanClusterServer.java

```diff
@@ -77,83 +79,93 @@ public static void main(String[] args) {
 
         // Abort if unexpected options
         if (cl.getArgs().length > 0) {
-            System.out.println("Unexpected option or arguments : " + cl.getArgList());
+            System.err.println("Unexpected option or arguments : " + cl.getArgList());
```

Comment:
it's not an error but it stop the the processing. for me it's the definition of a fatal error.